src/Microsoft.IdentityModel.Protocols/Configuration/StaticConfigurationManager.cs

```diff
@@ -44,7 +44,7 @@ public class StaticConfigurationManager<T> : IConfigurationManager<T> where T :
         /// <summary>
         /// Initializes an new instance of <see cref="StaticConfigurationManager{T}"/> with a Configuration instance.
         /// </summary>
-        /// <param name="configuration">Configuration of type <see cref="OpenIdConnectConfiguration"/> or <see cref="WsFederationConfiguration"/>.</param>
+        /// <param name="configuration">Configuration of type OpenIdConnectConfiguration or OpenIdConnectConfiguration.</param>
```

Comment:
Yes, when we refactored into different ns for WsFed and OIDC this would break. 
Probably best to say: 'Type that will contain configuration metadata'